README.rst

```diff
@@ -120,7 +120,8 @@ Testing
 ~~~~~~~
 
 After installation, you can launch the test suite from outside the
-source directory (you will need to have the ``pytest`` package installed)::
+source directory (you will need to have the ``pytest`` package installed
+(version >= 3.3.0))::
```

Comment:
maybe "you will need to have `pytest >= 3.3` installed"

Reply:
Done